packages/flutter_tools/lib/src/commands/build_flx.dart

```diff
@@ -12,6 +12,7 @@ import 'build.dart';
 class BuildFlxCommand extends BuildSubCommand {
   BuildFlxCommand({bool verboseHelp: false}) {
     usesTargetOption();
+    usesPackagesOption();
```

Comment:
I'd skip adding this function and just add the option directly.  If we want to use this from more than one command, we can refactor it to be shared later.